<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GS232ControllerGUI</class>
 <widget class="RollupContents" name="GS232ControllerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>378</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>GS-232 Rotator Controller</string>
  </property>
  <widget class="QWidget" name="controlsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Controls</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="controlLayout">
      <item>
       <widget class="ButtonSwitch" name="startStop">
        <property name="toolTip">
         <string>Start/stop controller</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/play.png</normaloff>
          <normalon>:/stop.png</normalon>:/play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="coord1Label">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Azimuth</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="coord1">
        <property name="toolTip">
         <string>Target azimuth in degrees</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>450.000000000000000</double>
        </property>
        <property name="value">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="coord1CurrentText">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Current azimuth in degrees</string>
        </property>
        <property name="text">
         <string>360.00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="coord2Label">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Elevation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="coord2">
        <property name="toolTip">
         <string>Target elevation in degrees</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="value">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="coord2CurrentText">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Current elevation in degrees</string>
        </property>
        <property name="text">
         <string>180.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="sources">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Target to track</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="track">
        <property name="toolTip">
         <string>Check to enable automatic tracking of azimuth and elevation from the specified channel</string>
        </property>
        <property name="text">
         <string>Track</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="targetName">
        <property name="toolTip">
         <string>Name of the target being tracked as indicated by the source channel / feature</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="dfmLine">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="dfmLayout">
      <item>
       <widget class="QToolButton" name="dfmTrack">
        <property name="text">
         <string>Track</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="dfmLubePumps">
        <property name="text">
         <string>Pumps</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="dfmBrakes">
        <property name="text">
         <string>Brakes</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="dfmDrives">
        <property name="text">
         <string>Drives</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="dfmShowStatus">
        <property name="toolTip">
         <string>Show the DFM status dialog</string>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QGridLayout" name="settingsGridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="azimuthOffsetLabel">
        <property name="text">
         <string>Azimuth offset</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSpinBox" name="precision">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Precision (number of decimal places) of azimuth, elevation and tolerance values</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="host">
        <property name="toolTip">
         <string>Hostname / IP address of computer to connect to</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="azimuthMax">
        <property name="maximum">
         <number>450</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="toleranceLabel">
        <property name="text">
         <string>Tolerance</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="serialPort">
        <property name="toolTip">
         <string>Name of serial port to use to connect to the rotator</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="azimuthOffset">
        <property name="toolTip">
         <string>Specify an offset angel in degrees that will be added to the target azimuth to correct for misalignment</string>
        </property>
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="azimuthMaxLabel">
        <property name="text">
         <string>Azimuth max</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="connectionLabel">
        <property name="text">
         <string>Connection</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="port">
        <property name="toolTip">
         <string>TCP port number to connect to</string>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="baudRateLabel">
        <property name="text">
         <string>Baud rate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="connection">
        <property name="toolTip">
         <string>The type of connection to use to the rotator</string>
        </property>
        <item>
         <property name="text">
          <string>Serial</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TCP</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="serialPortLabel">
        <property name="text">
         <string>Serial Port</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="tolerance">
        <property name="toolTip">
         <string>Tolerance in degrees</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="azimuthMin">
        <property name="maximum">
         <number>450</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="hostLabel">
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="elevationMaxLabel">
        <property name="text">
         <string>Elevation max</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="elevationMinLabel">
        <property name="text">
         <string>Elevation min</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="elevationMax">
        <property name="maximum">
         <number>180</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="azimuthMinLabel">
        <property name="text">
         <string>Azimuth min</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="elevationOffsetLabel">
        <property name="text">
         <string>Elevation offset</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="elevationMin">
        <property name="maximum">
         <number>180</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="baudRate">
        <property name="toolTip">
         <string>Serial port baud rate</string>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>230400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>460800</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="precisionLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Precision</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="protocol">
        <property name="toolTip">
         <string>Command protocol</string>
        </property>
        <item>
         <property name="text">
          <string>GS-232</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SPID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rotctld</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DFM</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="elevationOffset">
        <property name="toolTip">
         <string>Specify an offset angle in degrees that will be added to the target elevation to correct for misalignment</string>
        </property>
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="protocolLabel">
        <property name="text">
         <string>Protocol</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="coordinates">
        <property name="toolTip">
         <string>Coordinate system</string>
        </property>
        <item>
         <property name="text">
          <string>Az/El</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X/Y 85'</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X/Y 30'</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Coordinates</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
  <customwidget>
   <class>RollupContents</class>
   <extends>QWidget</extends>
   <header>gui/rollupcontents.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>startStop</tabstop>
  <tabstop>coord1</tabstop>
  <tabstop>coord2</tabstop>
  <tabstop>track</tabstop>
  <tabstop>sources</tabstop>
  <tabstop>targetName</tabstop>
  <tabstop>protocol</tabstop>
  <tabstop>connection</tabstop>
  <tabstop>serialPort</tabstop>
  <tabstop>baudRate</tabstop>
  <tabstop>host</tabstop>
  <tabstop>port</tabstop>
  <tabstop>azimuthOffset</tabstop>
  <tabstop>elevationOffset</tabstop>
  <tabstop>azimuthMin</tabstop>
  <tabstop>azimuthMax</tabstop>
  <tabstop>elevationMin</tabstop>
  <tabstop>elevationMax</tabstop>
  <tabstop>tolerance</tabstop>
 </tabstops>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
